docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -1096,7 +1096,7 @@ Remember to add the Authentication Rules with at least two Actions (example: Rol
 
 Moreover, don't forget to add LinkedIn as a *Source* from your connection profile definition, available from *Configuration -> Policies and Access Control -> Connection Profiles and Pages*.
 
-Also, LinkedIn requires a 'state' parameter for the authorization URL. If you modify it, make sure to add it at the end of your URL.
+NOTE: When testing LinkedIn OAuth2, use a different LinkedIn account to setup the application and to test the Source in the Captive Portal.
```

Comment:
Captive Portal -> captive portal

Reply:
Changed in https://github.com/inverse-inc/packetfence/pull/3759/commits/f9062580d423bedf4c0fb15cd9d84ef313626e46